modin/core/execution/ray/common/utils.py

```diff
@@ -206,7 +208,14 @@ def initialize_ray(
                 "object_store_memory": object_store_memory,
                 "_redis_password": redis_password,
                 "_memory": object_store_memory,
+                "runtime_env": {
+                    "env_vars": {
+                        "MODIN_LOG_MODE": LogMode.get(),
+                        "MODIN_LOG_FILE_SIZE": str(LogFileSize.get()),
+                    }
+                },
```

Comment:
I wonder if we should first merge https://github.com/modin-project/modin/pull/4603 and then augment the passed dictionary there (if logging is enabled) so that the configuration is checked for the case of externally-started clusters?